securedrop/source_templates/index.html

```diff
@@ -72,7 +72,7 @@ <h2 id="first-submission-heading">{{ gettext('First submission') }}</h2>
           <h2 id="return-visit-heading">{{ gettext('Return visit') }}</h2>
           <p>{{ gettext('Already have a codename? Check for replies or submit something new.') }}</p>
           <a href="{{ url_for('main.login') }}" class="btn secondary"
-            aria-label="{{ gettext('Log In') }}">
+            aria-label="{{ gettext('Log In Again Using A Previous Codename') }}">
```

Comment:
How would you feel about:

```suggestion
            aria-label="{{ gettext('Log In Using Your Codename') }}">
```

"Previous" makes sense from our system-level perspective.  But for a source I imagine it as "my" rather than "a previous" codename.

Reply:
I agree, this makes more sense when thinking from the perspective of a Source